cob_teleop/config/teleop_logitech.yaml

```diff
@@ -0,0 +1,97 @@
+#cob4-2
```

Comment:
yes, it is only an example. Actually as soon as I update cob_hardware_config we can delete it. Or keep a couple of examples, cob4-2 and an only-base cob4 (?)